packages/surveycto/src/Utils.js

```diff
@@ -0,0 +1,59 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  logResponse,
+  makeBasicAuthHeader,
+} from '@openfn/language-common/util';
+
+const addBasicAuth = (configuration = {}, headers) => {
+  const { username, password } = configuration;
+  if (username && password) {
+    Object.assign(headers, makeBasicAuthHeader(username, password));
+  }
+};
+
+const buildUrl = (configuration = {}, path) => {
+  const { servername, apiVersion = 'v1' } = configuration;
+  if (!servername) throw 'Please specify servername in your credentials';
+  return `https://${servername}.surveycto.com/api/${apiVersion}${path}`;
+};
+
+export const formDate = date => Math.floor(new Date(date).getTime() / 1000);
+
+export const prepareNextState = (state, response, callback) => {
+  const { body, ...responseWithoutBody } = response;
+  const nextState = {
+    ...composeNextState(state, response.body),
+    response: responseWithoutBody,
+  };
+
+  return callback(nextState);
+};
+
+export const requestHelper = (state, path, params, callback = s => s) => {
+  let {
+    body = {},
+    headers = { 'content-type': 'application/json' },
```

Comment:
Another note of feedback for `common.http` - we have to do this a lot and it also feels annoying. 

Oh, also, it's not really important by when GETing, we shouldn't include a content-type header (because there is no content)

Reply:
Oh, but actually when we use `requestHelper` at the moment, we ALWAYS pass a `content-type` header (as JSON or CSV).

So let's cut out the complication for now and just not set the default here